src/GenomeTrack.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * A track which displays a reference genome.
+ * @flow
+ */
+
+var React = require('react'),
+    types = require('./types');
+
+var GenomeTrack = React.createClass({
+  propTypes: {
+    range: types.GenomeRange,
+    basePairs: React.PropTypes.string
+  },
+  render: function(): any {
+    if (!this.props.range) {
+      return <EmptyTrack />;
+    }
+    var range = this.props.range;
```

Comment:
Move this above the previous `if` statement, and `s/this.props.range/range/`in that statement?